doc/local.satyh

```diff
@@ -70,11 +88,37 @@ let-rec math-of-type ty =
           let m = math-of-type ty in
             match ty with
             | TypeName(_) -> m
+            | Tuple(_)    -> m
+            | Record(_)   -> m
             | _           -> ${\paren{#m}}
         ))
       in
         Math.join ${\math-skip!(4pt)} (List.append mlst [text-in-math MathOrd (fun ctx -> read-inline ctx it)])
 
+  | Tuple(tylst) ->
+      let mlst =
+        tylst |> List.map (fun ty -> (
+          let m = math-of-type ty in
+            match ty with
+            | TypeName(_) -> m
+            | Tuple(_)    -> m
+            | Record(_)   -> m
+            | _           -> ${\paren{#m}}
+        ))
+      in
+      let m = Math.join ${,\math-skip!(2pt)} mlst in
+      ${\paren{#m}}
```

Comment:
In SATySFi, the syntax of product types is `τ_1 * τ_2 * … * τ_n`, not `(τ_1, τ_2, …, τ_n)` (unlike Haskell, Elm, etc.). Therefore, it seems more appropriate to display product types by:

```
Math.join ${\ast} mlst
```